packages/ocaml-variants/ocaml-variants.5.0.0+trunk/opam

```diff
@@ -52,7 +52,7 @@ build: [
 ]
 install: [make "install"]
 url {
-  src: "https://github.com/ocaml/ocaml/archive/trunk.tar.gz"
+  src: "https://github.com/ocaml/ocaml/archive/5.0.tar.gz"
```

Comment:
wow, the archive is actually called 5.0? No alpha/beta/...?

Reply:
It is the archive for the 5.0 branch (the version itself is 5.0.0+dev2-2022-06-08).